package.json

```diff
@@ -507,6 +507,7 @@
         "lolex": "^4.2.0",
         "marked": "^0.7.0",
         "mocha": "^6.2.0",
+        "mocha-junit-reporter": "^1.23.3",
```

Comment:
did you add this by hand or with npm? I think you'll need to use npm to do this in order to address the build issues.

Reply:
i did some npm command... the only build issues i'm seeing are on windows...